EIPS/eip-3525.md

```diff
@@ -0,0 +1,422 @@
+---
+eip: <to be assigned>
+title: <Versatile NFT Standard>
```

Comment:
It seems like a title like "Fractionalized NFT Standard" would more accurately represent the ERC you've proposed.

Reply:
Thanks for the suggestion, we had discussed many names for this standard include 'Fractionalized' one, but now seems this one is easier for understand. Now changed it to "Proposal for Fractionalized NFT Standard", however temporally left VFNT symbol unchanged for ease of modifications, once this one is accepted more, may think of changing the symbol to FNFT or sth. like that.